### PR TITLE
LF: Release LF 1.11

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
CHANGELOG_BEGIN

- [Daml-LF] Release LF 1.11
  - add suport for Generic Map
  - add support for choice observer
  - add generic comparison
  - add convert of contract ID to string (available only in off-ledger mode)
  - reduce size of dar using type interning

CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger-api/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.8.0
+1.10.0
```
Comment (cocreature) [inline ledger-api/VERSION:1]:
```suggestion
1.9.0
```
This is not tied to the SDK version, we just bump it when we make a change.

File: ledger/sandbox-classic/src/main/scala/platform/sandbox/SandboxServer.scala [hunk 2]
```diff
@@ -413,12 +413,10 @@ final class SandboxServer(
         )
       }
       if (config.seeding.isEmpty) {
-        // TODO https://github.com/digital-asset/daml/issues/7139
-        //  rephrase the message once LF 1.11 is released
         logger.withoutContext.warn(
-          s"""|'${Seeding.NoSeedingModeName}' contract IDs seeding mode is not compatible with the LF 1.11 languages or later which will be released soon.
+          s"""|'${Seeding.NoSeedingModeName}' contract IDs seeding mode is not compatible with the LF 1.11 languages or later.
```
Comment (cocreature) [inline ledger/sandbox-classic/src/main/scala/platform/sandbox/SandboxServer.scala:417]: My understanding was that @bame-da and you agreed to not have a default once 1.11 is stable and force users to choose. Is that still the plan? Perfectly fine not to include it in this PR.

File: daml-lf/language/daml-lf.bzl [hunk 3]
```diff
@@ -5,11 +5,11 @@
 # If we make a new LF release, we bump latest and once we make it the default
 # we bump stable.
 lf_stable_version = "1.8"
-lf_latest_version = "1.8"
+lf_latest_version = "1.11"
```
Comment (cocreature) [inline daml-lf/language/daml-lf.bzl:8]: have we already discussed one we flip the default in the compiler? one release after it’s stable like we used to or has that changed with the preview scheme?

**Review by cocreature — APPROVED** [review]
Thanks!

File: daml-lf/language/daml-lf.bzl (hunk 3 above)
Comment (remyhaemmerle-da) [reply daml-lf/language/daml-lf.bzl:8]: I am not quite sure. 
@sofiafaro-da what do you think ? 
Should we track that in an issue ?

File: ledger/sandbox-classic/src/main/scala/platform/sandbox/SandboxServer.scala (hunk 2 above)
Comment (remyhaemmerle-da) [reply ledger/sandbox-classic/src/main/scala/platform/sandbox/SandboxServer.scala:417]: I did not understand that. Let's sync with @bame-da and implement a fix (if necessary) in another PR.

File: daml-lf/language/daml-lf.bzl (hunk 3 above)
Comment (sofiafaro-da) [reply daml-lf/language/daml-lf.bzl:8]: I think the preview scheme is that we have one stable release before LF 1.11 becomes the default, and after that it becomes the default?

Comment (cocreature) [reply daml-lf/language/daml-lf.bzl:8]: sgtm so it will be the default in 1.11

Comment (sofiafaro-da) [reply daml-lf/language/daml-lf.bzl:8]: That's a nice coincidence -- that LF 1.11 becomes the default starting with SDK 1.11. Is there any chance we'd be able to keep new LF versions in sync with the SDK versions in which they become the default (by skipping a few version numbers ahead each time with the LF veresion)? Or is that a long shot?

**Pull request merged**